src/db_scripts/SQLScript_patch_6.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE customer 
+ADD COLUMN TotalNoVisits INT, 
```

Comment:
Change "TotalNoVisits" to "totalNoVisits".
The first character should be small letter.